docs/sources/shared/flow/reference/components/otelcol-queue-block.md

```diff
@@ -16,14 +16,14 @@ Name            | Type      | Description
 ----------------|-----------|----------------------------------------------------------------------------|---------|---------
 `enabled`       | `boolean` | Enables an in-memory buffer before sending data to the client.             | `true`  | no
 `num_consumers` | `number`  | Number of readers to send batches written to the queue in parallel.        | `10`    | no
-`queue_size`    | `number`  | Maximum number of unwritten batches allowed in the queue at the same time. | `5000`  | no
+`queue_size`    | `number`  | Maximum number of unwritten batches allowed in the queue at the same time. | `1000`  | no
 
 When `enabled` is `true`, data is first written to an in-memory buffer before sending it to the configured server.
 Batches sent to the component's `input` exported field are added to the buffer as long as the number of unsent batches doesn't exceed the configured `queue_size`.
 
 `queue_size` determines how long an endpoint outage is tolerated.
-Assuming 100 requests/second, the default queue size `5000` provides about 50 seconds of outage tolerance.
-To calculate the correct value for `queue_size`, multiply the average number of outgoing requests per second by the time in seconds that outages are tolerated.
+Assuming 100 requests/second, the default queue size `1000` provides about 10 seconds of outage tolerance.
+To calculate the correct value for `queue_size`, multiply the average number of outgoing requests per second by the time in seconds that outages are tolerated. Note that a very high figure can cause unexpected Out Of Memory (OOM) kills.
```

Comment:
```suggestion
To calculate the correct value for `queue_size`, multiply the average number of outgoing requests per second by the time in seconds that outages are tolerated. A very high value can cause Out Of Memory (OOM) kills.
```

Reply:
I don't see this suggestion applied. Generally we do not want to state `Note that` in the documentation. If it's a note, then we should make it a note... In this case a factual statement is enough.

Reply:
My bad, I failed to pull the changes here to my local. Commited!